PR першого розділу

yegor-poliakov opened the pull request:
Вніс багато правок до першого розділу. Сподіваюсь, розібрався з тим, щоб це правильно оформити в гітхабі

yegor-poliakov committed:
PR першого розділу

hedrok commented on 001.asc:
```diff
@@ -37,32 +36,32 @@ __...кров ллється літрами, і хтось викрикнув с
+шафа має по шість полиць і сягає майже стелі. Деякі полиці до краю заставлені
 книжками у твердій палітурці: фундаментальні науки, математика, історія тощо.
-На інших -- по два ряди наукової фантастики в мʼякій обкладинці.
+На інших -- у два ряди наукової фантастики в мʼякій обкладинці.
```
Мені "у два ряди наукової" дивно. Може "наукова фантастика в м’якій обкладинці у два ряди"?..

hedrok commented on 001.asc:
```diff
@@ -37,32 +36,32 @@ __...кров ллється літрами, і хтось викрикнув с
 Під задні підкладено старі коробки з-під серветок або дошки, щоб їх можна було
 бачити над передніми. Але й цього було не досить. Книжки заполонили
-столи й дивани, утворили невеличкі купи під вікнами.
+столи й дивани, книжки лежать невеликими стосами попід вікнами.
```
Оригінал "heap" перекладати як "стос" з моєї точки зору недоречно... Це саме купа, як це не дивно.

hedrok commented on 001.asc:
```diff
@@ -7,23 +7,22 @@ link:http://гпімр.укр/[Зміст]
 //rationality.
-Дисклеймер: Дж. К. Роулінґ має права на Гаррі Поттера, проте ніхто не має
-прав на методи раціональності.
+Дисклеймер: Права на Гаррі Поттера належать Дж. К. Роулінґ, права на методи раціональності не належать нікому.
```
Добре, тільки розбий на два рядки (після методи).

hedrok commented on 001.asc:
```diff
@@ -75,12 +74,12 @@ __...кров ллється літрами, і хтось викрикнув с
 //“This is absurd!” Michael said sharply. “They were at our wedding - they
 //visited for Christmas -”
--- Це абсурд! -- різко сказав Майкл, -- Вони були на нашому весіллі,
+-- Це нісенітниця! -- різко сказав Майкл, -- Вони були на нашому весіллі,
```
Мені здається професору більше пасує слово "абсурд". )

hedrok commented on 001.asc:
```diff
@@ -118,11 +117,11 @@ __...кров ллється літрами, і хтось викрикнув с
 Адже я __благала__ її. Роками я благала її. Лілі __завжди__ була красивішою за мене, і я...
-огидно ставилася до неї через це, а потім вона ще й отримала __магію__, можеш уявити собі,
-що я відчувала? І я __благала__ її використати трохи цієї магії на мені, щоб я теж могла
+жахливо ставилася до неї через це, а потім вона до всього отримала ще й __магію__, можеш уявити собі,
```
Якщо хочеш "огидно" -> "жахливо", то треба відновлювати довгу дискусію.
Якщо стисло: ми намагалися передати, що тепер Петунії самій огидно те, що вона робила. Спочатку я пропонував "злобно", але воно дивно звучить... Як точно передати "mean" ми не вигадали, "огидно" з поточних варіантів мені подобається найбільше.

yegor-poliakov replied:
мені не подобається "огидно", оскільки за першим прочитанням виникає асоціація, що вона ставилася до сестри, як до чогось огидного, але потім виникає ступор, що це мабуть щось не те,  і тоді перечитуєш по другому разу і розумієш, що малося на увазі (що ставлення заслуговує на огиду). "жахливо" у цьому сенсі викликає більш однозначну асоціацію, тому що жахливе ставлення зазвичай є несправедливим. але загалом найточніший переклад mean тут буде "deliberately unkind", тобто жорстоко. відповідно пропоную сказати "була до неї жорстока через це". Наполягати не буду, втім, якщо вам нормально звучить "огидно", то не будемо піднімати закритий тред

ElviraIzmailova replied:
Вона і ставилася до неї, як до чогось огидного, називала фріком.
Але я згодна з Єгором тут.

ElviraIzmailova replied:
@yegor-poliakov ага

hedrok commented on 001.asc:
```diff
@@ -162,19 +161,19 @@ __...кров ллється літрами, і хтось викрикнув с
 //what her magic brought her in the end -”
--- Хай там як, -- промовила Петунія тихим голосом, -- вона здалася. Вона сказала мені,
-що це небезпечно, а я відповіла, що мені вже байдуже. І я випила це зілля, і нездужала
+-- Хай там як, -- промовила Петунія тихо, -- вона здалася. Лілі попередила,
```
Тоді напевно вже "тихо промовила Петунія"?..

hedrok commented on 001.asc:
```diff
@@ -162,19 +161,19 @@ __...кров ллється літрами, і хтось викрикнув с
 //change your diet -”
--- Рідна, -- лагідно сказав Майкл, -- ти захворіла, і доки відпочивала в ліжку,
-набрала ваги, а шкіра очистилася сама собою. Чи хвороба змусила тебе
+-- Рідна, -- лагідно сказав Майкл, -- ти захворіла, і доки відновлювалася в ліжку, то
```
Чому "відовлювалася"?..

yegor-poliakov replied:
мені не дуже подобається це речення, бо коли ти хворий, то ти не просто відпочиваєш у ліжку, а саме відновлюєшся. але ти маєш рацію, що воно теж коряво звучить. повертаю відпочивала

hedrok commented on 001.asc:
```diff
@@ -267,21 +266,21 @@ __...кров ллється літрами, і хтось викрикнув с
-прошарку голодних студентів. Гаррі завжди заохочували вивчати все, що його цікавило,
-купували всі книжки, що були йому до вподоби, оплачували будь-які математичні чи наукові
-змагання, в яких він приймав участь. Йому в межах припустимого давали все, що він захоче,
+Гаррі гірко скривився. Батько ставився до нього добре, напевно краще, ніж більшість
```
Я б "До нього ставилися добре" залишив. В оригіналі "He was treated well".

yegor-poliakov replied:
в оригіналі treated не має числа, відповідно неясно, чи йдеться про батьків чи про батька. і тоді виходить плутанина: якщо ми пишемо, що краще, ніж більшість татусів, то до чого тут мама? і так само далі по тексту говориться про професора Оксфорду. тому прошу все ж залишити тут "Батько ставився до нього добре"

hedrok replied:
Добре, я не проти.

hedrok commented on 001.asc:
```diff
@@ -267,21 +266,21 @@ __...кров ллється літрами, і хтось викрикнув с
+Гаррі гірко скривився. Батько ставився до нього добре, напевно краще, ніж більшість
+генетичних тутусів ставляться до власних дітей. Гаррі були відправляли до найкращих
+початкових шкіл, а коли з цим не склалося, то знайшли репетиторів з невичерпного
+резерву голодних студентів. Гаррі завжди заохочували вивчати все, що його цікавило,
```
Чому "прошарку" -> "резерву"?

yegor-poliakov replied:
прошарок - це термін з соціології, коли ми позначаємо окрему соціальну групу, тоді як у цьому разі йдеться про group of people available for some purpose. резерв теж не найбільш вдале слово, пропоную так:
"а коли з цим не склалося, то знаходили репетиторів серед нескінченної юрби голодних студентів"

ElviraIzmailova replied:
Юрба - супер.

hedrok replied:
:+1:

hedrok commented on 001.asc:
```diff
@@ -267,21 +266,21 @@ __...кров ллється літрами, і хтось викрикнув с
+змагання, в яких він брав участь. У розумних межах йому давали все, що він захоче,
 окрім, можливо, бодай дещиці поваги. Від доктора наук, що читає біохімію в Оксфорді,
-навряд чи можна вимагати, щоб той прислухався до порад маленького хлопчика. Він, звісно,
+навряд чи можна вимагати, щоб той дослухався до порад маленького хлопчика. Але він, звісно,
```
Мені здається, що"Але" тут зайве.

hedrok commented on 001.asc:
```diff
@@ -293,19 +292,17 @@ __...кров ллється літрами, і хтось викрикнув с
 //your father. I want my husband to, to listen to his wife who loves him, and
 //trust her just this once -”
-Мати подивилася на нього вниз і всміхнулася.
+Мати подивилася на нього і всміхнулася.
```
Чому?..

yegor-poliakov replied:
тут йдеться або про "подивилася на нього зверхньо", що зазвичай і означає look down, або ж про те, що їй довелося подивитися вниз, бо Гаррі дуже нижчий. Перший варіант не підходить, другий дуже дивно звучить:  "Єгор подивився на Кирила вниз і всміхнувся". Я би таке вживав тільки у випадку, якби ми в підвал вантажили картоплю. Тому пропоную просто викинути "вниз" як не дуже значущу деталь.

ElviraIzmailova replied:
Мені здається, треба все ж таки лишити, хоча не принципово.

hedrok replied:
Я за такий варіант.

ElviraIzmailova replied:
Мене цілком влаштовує варіант Єгора ^^

hedrok commented on 001.asc:
```diff
@@ -357,7 +354,7 @@ __...кров ллється літрами, і хтось викрикнув с
 //explain how it arrived at the letterbox without a stamp. A little insanity was
 //far, far less improbable than the universe really working like that.
-Це мало б бути безперечною ознакою того, що мама жартує, бреше або збожеволіла,
+Це мало б бути беззаперечною ознакою того, що мама жартує, обманює або збожеволіла,
```
Чому "обманює"?..

yegor-poliakov replied:
тому що "бреше" досить грубе і просторічне слово, яке навряд чи доречне у дискурсі про маму, яку Гаррі любить

hedrok replied:
Я не вважаю "брехати" ані грубим, ані просторічним.
брехати = казати неправду = lie.
Обманювати - просто ширше значення.
І не забувай "у порядку зростання жахливості" :)
Але сподіваюся тут Ельвіра скаже своє слово )

hedrok replied:
Як скаже Ельвіра, так і зробимо.

yegor-poliakov replied:
@ElviraIzmailova

ElviraIzmailova replied:
Я згодна з тим, що це розмовне слово і воно доволі грубе, але вважаю його тут доречним, вибачай, Єгор. 
Я за "бреше". Тому що "обманює" занадто звучить наче це вже довготривала продумана брехня. Може тоді просто "каже неправду"? Якщо ні, то "бреше".

hedrok commented on 001.asc:
```diff
@@ -384,35 +381,34 @@ __...кров ллється літрами, і хтось викрикнув с
 так, професор Гоґвортсу зʼявиться, махне чарівною паличкою, і буде магія. Ця дивна
-впевненість не намагалася захистити себе від спростування -- не вигадувала
-завчасно виправдань на випадок, якщо професор не прийде, або зможе
+впевненість не намагалася захистити себе від спростування, не вигадувала
```
Чому кома? Далі пояснення. Можна двокрапку, якщо хочеш.

yegor-poliakov replied:
двокрапка тут не пасує, поверну тире. для мене просто неочевидно, що друге речення є уточненням до першого, а не парафраз

hedrok commented on 001.asc:
```diff
@@ -384,35 +381,34 @@ __...кров ллється літрами, і хтось викрикнув с
 //hypothesis is to go and test it.
-Гаррі уявно знизав плечима. Пласку металеву пластину на двері штовхають, ручку
-дверей тягнуть, а гіпотезу, яку можна перевірити, перевіряють.
+Гаррі подумки знизав плечима. Двері штовхають або тягнуть за ручку, 
```
Вибач, але :-1:
Ну не передає ж зовсім.
Можна спробувати вигадати інші дві очевидні речі, наприклад: "У дзвоник дзвонять, шкарпетки одягають на ноги, а...".
Має бути перелічення двох речей і двох очевидних дій з ними, щоб потім показати, що "гіпотезу, яку можна перевірити" - "перевіряють".

(до "подумки" претензій немає)

yegor-poliakov replied:
давай думати. одне з важливих правил перекладу - читач має розуміти, про що йдеться, якщо тільки це не заплановане автором нерозуміння. пласка металева пластина на дверях в Україні зустрічається тільки в американському посольстві, тому треба знайти якийсь адекватний приклад. загалом воно і в оригіналі досить незграбне, бо в реченнях про двері йдеться про те, як вони спрацьовують, а в реченні про гіпотезу - що з нею природньо треба робити.
Пропоную так: "Гаррі подумки знизав плечима. У дзвоник дзвонять, гвіздок забивають, а гіпотезу, яку можна перевірити, перевіряють".

hedrok replied:
СУМ знає "гвіздок"... Я не знаю "гвіздок" )
Я гадав, що він "цвях". Якщо все одно - став "цвях", якщо ні - чекаємо на Ельвіру.
Решта подобається.

hedrok commented on 001.asc:
```diff
@@ -384,35 +381,34 @@ __...кров ллється літрами, і хтось викрикнув с
 // 78
 //He took a piece of lined paper from his desk, and started writing.
-Він взяв зі стола аркуш паперу в лінійку і почав писати.
+Він узяв зі столу аркуш паперу в лінійку і почав писати.
```
http://rodovyi-vidminok.wikidot.com/s
Там, на жаль, не розписано, коли "-у", а коли "-а". Але я вживаю так:
Коли йдеться про їжу, то "-у" (nтак само як "сніданок" -> "сніданку").
Коли про меблі, то "-а" (так само як стілець -> стільця)

hedrok commented on 001.asc:
```diff
@@ -422,52 +418,51 @@ __Шановна заступнице директора__
-Гаррі зупинився поміркувати. Потім взяв інший аркуш і висунув ще міліметр графіту з
-механічного олівця. Тут потрібен охайний почерк.
+Гаррі зупинився поміркувати, взяв інший аркуш і висунув ще міліметр графіту з
+механічного олівця. Тут краще писати охайно.
```
Я проти. "This called for careful calligraphy" -> "Тут краще писати охайно" це вже зовсім далеко.

hedrok commented on 001.asc:
```diff
@@ -422,52 +418,51 @@ __Шановна заступнице директора__
 // 82
 //__Or Whomsoever It May Concern:__
-__Або будь-хто, кого це може стосуватися:__
+__чи будь-хто, кого це може стосуватися:__
```
@roadischosen тут виправляв "Чи" -> "Або". І мені здається, що потрібна все ж таки велика літера.

yegor-poliakov replied:
"або" виправив на "чи" з міркувань милозвучності: тут і так багато голосних, які ще й складаються в "абобудь".
англійською всі слова з великої літери, як наче це заголовок, відповідно ми або дублюємо всі слова великою, або пишемо "або"/"чи" з маленької літери

hedrok replied:
Якщо тобі більш-менш все одно, поверни "Або" - @roadischosen вважає, що в них різний зміст. Я не розбирався ще, в чому різниця, але він виправляв не з питань милозвучності, тому я б довірився тут йому.

yegor-poliakov replied:
почитав [тут](http://forum.slovnyk.ua/index.php?showtopic=2499), звучить переконливо. у цій ситуації, правда, бачу, що можна відстоювати і варіант з "чи" (Гаррі не впевнений, хто саме йому потрібен), але не буду і повертаю "Або"

hedrok commented on 001.asc:
```diff
@@ -422,52 +418,51 @@ __Шановна заступнице директора__
-користуватися. Мій батько вкрай скептично налаштований. Сам я не впевнений. Я
-також не знаю, де можна придбати хоч щось з книжок чи приладдя, що були
-вказані у вашому листі.__
+__Я надзвичайно зацікавлений навчатися в Гоґвортсі, якщо таке місце справді
```
"за умови" -> "якщо" не погоджуюся. Оригінал "conditional on", там же не "if" стоїть.
"Я вкрай зацікавлений у навчанні" мені подобається більше, ніж "Я надзвичайно зацікавлений навчатися". Але тут легко прийму рішення більшості.
Чому "дійсно" -> "справді"? Тут використання "дійсно" мені здається доречним.

yegor-poliakov replied:
якщо ніхто не заперечує, то хотів би залишити "Я надзвичайно зацікавлений навчатися в Гоґвортсі за умови, що таке місце дійсно існує".

ElviraIzmailova replied:
Гарно.

hedrok commented on 001.asc:
```diff
@@ -422,52 +418,51 @@ __Шановна заступнице директора__
-також не знаю, де можна придбати хоч щось з книжок чи приладдя, що були
-вказані у вашому листі.__
+__Я надзвичайно зацікавлений навчатися в Гоґвортсі, якщо таке місце справді
+існує. Моя мати Петунія каже, що знає про магію, проте сама нею не володіє. 
```
"Тільки" є в оригіналі: "Only" і передає, що далі буте описано, в чому проблема Гаррі з тим, щоб навчатися, хоча він хоче.
Я вважаю, що його варто залишити.

yegor-poliakov replied:
мене збиває багатозначність використання тільки: "от тільки моя мама ...., а тато, а я..." або "тільки моя мама вірить...". але нехай буде, повертаю його

hedrok commented on 001.asc:
```diff
@@ -422,52 +418,51 @@ __Шановна заступнице директора__
+__Я надзвичайно зацікавлений навчатися в Гоґвортсі, якщо таке місце справді
+існує. Моя мати Петунія каже, що знає про магію, проте сама нею не володіє. 
+Мій батько налаштований вкрай скептично. Сам я вагаюся. Я
+також не знаю, де можна придбати хоч щось з книжок чи приладдя, перелічених у Вашому листі.__
```
"перелічених" -> "що перелічені".
Ну і "Вашому" знову з маленької треба.

yegor-poliakov replied:
перелічені вставив, щодо Ви - як вирішимо вище

hedrok commented on 001.asc:
```diff
@@ -480,9 +475,9 @@ __Гаррі Джеймс Поттер-Еванс-Веррес.__
-Гаррі додав їхню поточну адресу, склав листа і поклав до конверта, що він
-адресував до Гоґвортсу. Подальші роздуми змусили його взяти свічку й накрапати
-воску на клапан конверту, щоб запечатати його. Потім, кінчиком ножа, він вирізав на печатці ініціали
+Гаррі додав їхню чинну адресу, склав листа і поклав до конверта, 
```
З цього абзацу готовий погодитися лише зі зміною "взяти свічку й накрапати воску" -> "накрапати воску зі свічки".
Решта не подобається:

Чому "чинна адреса"?.. Оригінал "current address". З СУМ:
http://sum.in.ua/s/chynnyj 1. Який діє за певних умов, має юридичну силу.
http://sum.in.ua/s/potochnyj 1. Який є, існує, відбувається, має місце тепер, у цей час

"що він адресував" -> "адресованого" не подобається напевно через особистий смак
"Подальші роздуми змусили його" набагато ліпше передає оригінал з моєї точки зору.

hedrok commented on 001.asc:
```diff
@@ -492,8 +487,8 @@ __Гаррі Джеймс Поттер-Еванс-Веррес.__
-Потім він відчинив двері й спустився сходами. Батько сидів у кімнаті й читав книжку
-з вищої математики, щоб показати, який він розумний; мати готувала на
+Потім він відчинив двері й спустився сходами. Батько сидів у кімнаті й читав 
+складну математичну книжку, щоб показати, який він розумний; мати готувала на
```
Мені @roadischosen довів, що "higher mathematics" - це "вища математика":
https://en.wikipedia.org/wiki/Further_Mathematics
https://uk.wikipedia.org/wiki/Вища_математика

hedrok commented on 001.asc:
```diff
@@ -532,10 +527,10 @@ __не сперечалися__ вони якимось чином набага
-була чимось ганебним, в неї могли вірити лише дурні; якби його батько опустився до
-__перевірки__ цієї гіпотези, або навіть просто __подивився__, як її перевіряють, то це ніби б
-__повʼязало__ його з усім цим...
+Батько ледь похитав головою і продовжив читати. __Звісно,__ -- подумав Гаррі. -- Магія
```
Тоді вже й лапки треба ставити... Я не оформлював з тире, бо в оригіналі немає відповідних знаків...

hedrok commented on 001.asc:
```diff
@@ -622,20 +618,20 @@ __Ні. Я кращий за тата. Я використаю науковий
 //Hardly even thinking at this point, Harry gave over his envelope.
 Зморщена рука простягнулася над парканом, долоня розкрита в очікуванні.
-Майже бездумно, Гаррі віддав свого листа.
+Збентежений, Гаррі віддав листа.
```
Чому "збентежений"?.. Оригінал "hardly even thinking".

hedrok replied:
Не переконав. Чекаємо на Ельвіру. (з моєї точки зору все ж таки важливо вказати, що це було не обмірковане рішення, тобто він не продумав можливі наслідки, а просто віддав, бо попросили).

yegor-poliakov replied:
давай залишимо тоді "Майже бездумно, Гаррі віддав листа".

hedrok commented on 001.asc:
```diff
@@ -648,13 +644,13 @@ __Ні. Я кращий за тата. Я використаю науковий
 Потім голос хлопчика спокійно й тихо сказав:
 
--- Що.
+-- Що?
```
В оригіналі крапка.

yegor-poliakov replied:
Вона-то крапка, але це явно питання. Якщо принципово, то хай буде крапка, але мені здається, що це питання і що хай воно краще й буде питанням)

ElviraIzmailova replied:
Це дуже слушна думка (:

hedrok replied:
Мені настільки не подобається цей варіант, що я аж запропоную декілька альтернатив:
-- Що за.
-- Якого.
Але якщо ви вдвох вважаєте, що "Що?" найкраще передає, то що ж поробиш - виправляйте.

yegor-poliakov replied:
якщо Ельвіра після цього ставить лайк, то ставимо знак питання)

ElviraIzmailova replied:
Мені подобається "Якого...". Саме з крапками. 
Але якщо залишите "що", то тоді зі знаком питання. 
Якщо не хочеться натякати на лайку, можна просто іншим чином висловити розгубленість, просто написати "Еееем" чи щось таке. 

Дивно, цей комент загубився.

yegor-poliakov replied:
ставлю "Якого..."

yegor-poliakov commented on 001.asc:
```diff
@@ -422,52 +418,51 @@ __Шановна заступнице директора__
 //Petunia Evans-Verres, and her husband, Michael Verres-Evans.__
-__Я нещодавно отримав вашого листа про вступ до Гоґвортсу, що було адресовано
-``містерові Г. Поттеру'' Можливо вам невідомо, що мої генетичні батьки, Джеймс
+__Я нещодавно отримав Вашого листа про вступ до Гоґвортсу, що було адресовано
```
те, як написаний лист, свідчить, що він звертається до Макґонеґел: він згадує її по імені, тож це вона підписалася під листом до нього, і відповідає він передусім їй, а зворот про "будь-кого іншого" вживає для перестраховки. в формальному листі, мені здається, все ж варто зберегти велику літеру. вони ж там навіть між дітьми по прізвищу звертаються

yegor-poliakov replied:
Пономарів [пише](http://www.bbc.co.uk/blogs/ukrainian/ponomariv/2010/03/post-15.html), що Леся Українка писала з великої.
Ось також [лист Кримського до Грінченка](http://zbruc.eu/node/57500)

yegor-poliakov replied:
@ElviraIzmailova

ElviraIzmailova replied:
З великої.
а) Професорка займає досить високу посаду в елітному навчальному закладі, вона заслуговує на пошану :3 
б) Гаррі трохи страшно і він хоче показати себе в ліпшому світлі, якомога серйознішим і офіційним, він навіть крапає воск і вирізає ініціали(!). Тому мені здається, що якщо є якась необов'язкова опція зробити листа більш офіційним і пошанним - він її використає.
в) офіційного правила наче немає, але традиція наче є, тому опція такая існує.

yegor-poliakov committed:
Update 001.asc

ElviraIzmailova commented on 001.asc:
```diff
@@ -507,9 +503,9 @@ __не сперечалися__ вони якимось чином набага
 //His mother turned from the kitchen sink to stare at him, looking shocked. “I -
 //I don’t know, I think you just have to own a magic owl.”
-Його мати відволіклася від раковини щоб поглянути на нього. Вона була шокована.
+Петунія повернулася від раковини і розгублено поглянула на нього.
```
"відвернулася" тоді. Та й "відвлоіклася" мені більше подобається.

ElviraIzmailova commented on 001.asc:
```diff
@@ -532,10 +528,10 @@ __не сперечалися__ вони якимось чином набага
+Батько ледь похитав головою і продовжив читати. __Звісно,__ подумав Гаррі. Магія
+-- це щось ганебне, в неї можуть вірити лише дурні. Якби батько опустився до
+__перевірки__ цієї гіпотези або навіть просто __подивився__, як її перевіряють, то цим ніби 
+__повʼязався__ б з усім цим...
```
цим - цим
І __повʼязався__ якось дивно.

yegor-poliakov replied:
"Якби батько опустився до __перевірки__ цієї гіпотези або навіть просто __подивився__, як її перевіряють, то ніби й сам __увʼязався__ б з усім цим..."
http://sum.in.ua/s/vv.jazuvatysja

ElviraIzmailova replied:
Супер!

ElviraIzmailova commented on 001.asc:
```diff
@@ -555,17 +551,18 @@ __повʼязало__ його з усім цим...
 // 97
 //Harry took a deep breath, and raised the envelope into the air.
-Гаррі вдихнув і підняв конверт догори.
+Гаррі глиобко вдихнув і підняв конверт догори.
```
Опечатка.

ElviraIzmailova commented on 001.asc:
```diff
@@ -577,7 +574,7 @@ __Ні. Я кращий за тата. Я використаю науковий
 // 101
 //“Letter -” Harry said, but it actually came out as more of a whispered croak.
--- Лист... -- сказав Гаррі, проте насправді вийшло щось на кшталт тихого хрипіння.
+-- Лист... -- спробував вигукнути Гаррі, але звук вийшов подібним більше на шепітливий хрип.
```
"більше" треба прибрати чи переставити.

ElviraIzmailova commented on 001.asc:
```diff
@@ -446,17 +447,17 @@ __Я нещодавно отримав Вашого листа про вступ
 //my own family it would be extremely helpful.__
 __Мати згадувала, що Ви відправили представника Гоґвортсу до Лілі Поттер (тоді
-Лілі Еванс), щоб довести її родині, що магія існує, і, я гадаю, для
+Лілі Еванс), щоб продемонструвати її родині, що магія існує, і, я гадаю, для
```
Я вже не розумію, хто що куди виправляв, але це місце дивне.
Можна продемонструвати магію, можна довести існування магії, але продемонструвати існування.. Це якось дивно.

yegor-poliakov replied:
Не бачу проблеми з демонстрацією існування, саме це ж має на увазі Юдковськи: продемонструвати, що магія є справжньою, тобто що вона існує. українською просто буде дивно звучати "продемонструвати, що магія справжня", бо тоді виходить, наче є якась підробна магія.
Давай тоді так: "Мати згадувала, що Ви присилали представника Гоґвортсу до Лілі Поттер (тоді Лілі Еванс), щоб продемонструвати її родині, що магія справді існує, і, припускаю, щоб допомогти Лілі з придбанням її шкільних речей".
*Ви потім будемо виправляти, коли зупинимося на якомусь одному варіанті

hedrok commented on 001.asc:
```diff
@@ -37,32 +37,32 @@ __...кров ллється літрами, і хтось викрикнув с
 Під задні підкладено старі коробки з-під серветок або дошки, щоб їх можна було
 бачити над передніми. Але й цього було не досить. Книжки заполонили
-столи й дивани, утворили невеличкі купи під вікнами.
+столи й дивани, книжки лежать невеликими стосами попід вікнами.
```
Ми ж домовилися про:
> тоді давай "книжки лежать невеликими купами попід вікнами"

yegor-poliakov replied:
ще не встиг відреагувати, сорі. я не вношу правки нишком)

hedrok replied:
Просто коментар зник - написав, щоб не загубилося.

yegor-poliakov committed:
Update 001.asc

hedrok commented on 001.asc:
```diff
@@ -320,22 +318,22 @@ __...кров ллється літрами, і хтось викрикнув с
-Тепер батьки знову перейдуть до однієї з __тих__ суперечок, коли мати
-намагалася змусити батька почуватися винним, а той намагався змусити матір почуватися 
+Тепер батьки знову почнуть одну з __тих__ суперечок, коли мати
+намагається змусити батька почуватися винним, а той намагаєтьсяся змусити її почуватися 
```
одрук: "намагаєтьсяся" -> "намагається"

hedrok commented on 001.asc:
```diff
@@ -384,35 +382,34 @@ __...кров ллється літрами, і хтось викрикнув с
 //hypothesis is to go and test it.
-Гаррі уявно знизав плечима. Пласку металеву пластину на двері штовхають, ручку
-дверей тягнуть, а гіпотезу, яку можна перевірити, перевіряють.
+Гаррі подумки знизав плечима. Гаррі подумки знизав плечима. У дзвоник дзвонять, 
```
Два рази "Гаррі подумки знизав плечима"

ElviraIzmailova commented on 001.asc:
```diff
@@ -7,23 +7,23 @@ link:http://гпімр.укр/[Зміст]
 //rationality.
-Дисклеймер: Дж. К. Роулінґ має права на Гаррі Поттера, проте ніхто не має
-прав на методи раціональності.
+Дисклеймер: Права на Гаррі Поттера належать Дж. К. Роулінґ, права на методи 
```
Прибери "у" з прізвища Дж. К., будь ласка, ми забули (:

ElviraIzmailova replied:
Мені подобається "Якого...". Саме з крапками. 
Але якщо залишите "що", то тоді зі знаком питання. 
Якщо не хочеться натякати на лайку, можна просто іншим чином висловити розгубленість, просто написати "Еееем" чи щось таке.

yegor-poliakov committed:
Контрольний коміт розділу 1

Прийняв усі пропоновані правки.
Зробив також ще трохи нових з пунктуацією:
- по комах при прямій мові: у реченнях форми "Не може бути, – сказав Гаррі. – Я краще з’їм свою паличку", треба кома, а не крапка після "Не може бути". Аналогічно у рефеннях форми "Не може бути, – сказав Гаррі, – щоб я був не поїхав до Гоґвортсу" друга частина пишеться після коми, якщо покликання на того, хто говорить, розриває речення. Це так за правописом (http://r2u.org.ua/pravopys/pravXXI/136.html).
- в кінці речення, де Юдковськи позначає обрив речення простим дефісом, пропоную поставити крапки (http://litopys.org.ua/pravopys/rozdil5.htm)
Також:
- прибрав кому у виразі "що мої генетичні батьки, Джеймс Поттер і Лілі Поттер", оскільки якщо це уточнення, то треба виділяти з обох боків, або ж прибрати кому, що і пропоную зробити
- замінив "б" -> "би" у виразі "Це мало б бути беззаперечною ознакою того"
Якщо заперечень немає, то коміт можна об’єднувати, якщо ж є наполягання залишити дефіси, а не крапки, то поверну як було

yegor-poliakov commented on 001.asc:
```diff
@@ -68,26 +68,26 @@ __...кров ллється літрами, хтось викрикує сло
--- Я сказала їм, що тобі не слід знати. -- прошепотіла Петунія, -- Проте це правда.
-Я бачила речі-
+-- Я сказала їм, що тобі не слід знати, -- прошепотіла Петунія, -- Проте це правда.
+Я бачила речі...
```
Тут також ще треба крапку після "прошепотіла Петунія"

yegor-poliakov committed:
Update 001.asc

ElviraIzmailova reviewed (approved):
Всі поточні питання вирішено.

PerfectGirl1 commented:
Это мама егора?
